guides/common/modules/con_deploying-project-on-aws.adoc

```diff
@@ -1,6 +1,7 @@
-[id="scenarios-for-{project-context}-on-aws"]
-= Scenarios for {Project} on AWS
+[id="deploying-{project-context}-on-aws"]
+= Deploying {Project} on AWS
 
+You can run {ProjectServer}, {SmartProxyServers}, and hosts on your Amazon Web Services (AWS) environment.
```

Comment:
```suggestion
You can run {ProjectServer}, {SmartProxyServers}, and hosts in your Amazon Web Services (AWS) environment.
```
@bangelic What preposition is appropriate here? :laughing:

Reply:
I'm not Brian but I agree with your suggestion.

Reply:
Ack. Makes sense to use "in" here :+1: